packages/rebound/meta.yaml

```
@@ -0,0 +1,16 @@
+package:
+  name: rebound
+  version: 3.19.4
+build:
+  exports: requested
+source:
+  url: https://files.pythonhosted.org/packages/78/37/95ab84dfb72a37d78420ffa1ee087dfbb6f1098117435e2191f2deff9067/rebound-3.19.4.tar.gz
+  sha256: b705235c6619b4be5368faaa07a865e6d44f9856b9b31721d1032ebe0f2199ec
+test:
```

Because you need `numpy` in your test, I think you technically need to add
```suggestion
requirements:
  run:
    - numpy
test:    
 ```
here (there's no way to specify that a requirement is only for testing at this time AFAIK). Otherwise, the test would not be able to run in a partial build that doesn't include `numpy`. But I think the CI tests will work either way, because they build `numpy`.